### ISSUE TITLE
Use defined fields as default

### ISSUE THREAD

**Issue by CinKon** [revolver-app/vuex-orm#25]
Is there a way, to use defined fields as default values, if the `create` or `insert`-method doesn't contain all keys?

Let me give you an example:

Model:
```javascript
export default class User extends Model {
  // This is the name used as module name of the Vuex Store.
  static entity = 'user'

  static primaryKey = 'id'

  // List of all fields (schema) of the post model. `this.attr` is used
  // for the generic field type. The argument is the default value.
  /* eslint-disable camelcase */
  static fields () {
    return {
      id: this.attr(null),
      created: this.attr(null),
      lastLogin: this.attr(null),
      roleId: this.attr(null),
      roleName: this.attr(null),
      username: this.attr(''),
      email: this.attr(''),
      gender: this.attr(null),
      firstname: this.attr(''),
      surname: this.attr(''),
      street: this.attr(''),
      zip: this.attr(''),
      city: this.attr(''),
      mobile: this.attr(''),
      phone: this.attr(''),
      online: this.attr(false),
      inactive: this.attr(false)
    }
  }
}
```

Insert:

```javascript
const users = [{
  id: '144432',
  username: 'foobar'
}]

store.dispatch('entities/user/create', { data: users })
```

With this, the store only contains `id` and `username` for this entity, but I'd need all the other keys.

**Comment by kiaking** [comment]
YES, recently I came across this issue too. Thank you for bringing up.

The reason why this happens is because default value of Model get sets when instantiating Model but not when creating data in to store. So maybe as you may already know if you use getters to fetch the data, all the field will be there within model instance with its default value regardless of the store state.

Which means the current architecture decouples the Model and Module. You can actually save "any" data to the store. The only thing Modules care is the relationship attribute. This gives some flexibility for example you can create totally different schema between Model and Module. This might be maybe useful when you have some data you want to use for getters `when` method but you don't need that in Model instance.

However, I think your use case or needs makes more sense. So I guess I'll add this feature. If there are no corresponding key to the model when saving data through store action, add that key with default value.

For now, you may workaround this issue by doing something like this.

```js
// Create empty Model. This will fill all the fields with its default value.
const user = new User()

// $toJson method on the model converts all the fields into plain object.
const defaultData = user.$toJson()

// Data you would want to save.
const data = { id: '144432', username: 'foobar' }

// Merge 2 data and make new one.
const mergedData = { ...defaultData, ...data }

// Save the data!
store.dispatch('entities/users/create', { data: mergedData }
```

Well obviously you could make above code shorter but still very tedious 😞 

Please stay tuned for the fix!

**Comment by kiaking** [comment]
@CinKon I made PR #33 to solve this issue but I want to confirm the behavior with you. Would you mind looking at the PR and let me know what you think?

**Comment by kiaking** [comment]
This issue is fixed with Vuex ORM 0.13.2.

@CinKon Thank you for your support as always!
